python/cuml/tests/test_metrics.py

```diff
@@ -1104,6 +1107,15 @@ def prep_dense_array(array, metric, col_major=0):
         return np.asfortranarray(array) if col_major else array
 
 
+@pytest.mark.filterwarnings(
+    "ignore:X was converted to boolean for metric russellrao:UserWarning"
+)
+@pytest.mark.filterwarnings(
+    "ignore:Y was converted to boolean for metric russellrao:UserWarning"
+)
+@pytest.mark.filterwarnings(
+    "ignore:Data was converted to boolean for metric russellrao:sklearn.exceptions.DataConversionWarning"
+)
```

Comment:
The warnings here seem like something we could remove by amending the tests.

Reply:
I think there are a ton of warnings that we can (easily) address rather than ignore. I tried to keep this PR focused and explicitly _not_ address any of them.